src/components/OpprettPlanPage/FyllUtPlanSteg/form/hooks/useOppfolgingsplanLagring.ts

```diff
@@ -0,0 +1,121 @@
+import { startTransition, useActionState, useTransition } from "react";
+import { useParams } from "next/navigation";
+import { OppfolgingsplanForm } from "@/schema/oppfolgingsplanFormSchemas";
+import { ferdigstillPlanServerAction } from "@/server/actions/ferdigstillPlan";
+import { lagreUtkastServerAction } from "@/server/actions/lagreUtkast";
+
+export type LagreUtkastActionState = {
+  isLastUtkastSaveSuccess: boolean;
+  utkastLastSavedTime: Date | null;
+  lastSavedValues: OppfolgingsplanForm | null;
+};
+
+export default function useOppfolgingsplanLagring({
+  initialFormValues,
+  initialLastSavedTime,
+}: {
+  initialFormValues: OppfolgingsplanForm;
+  initialLastSavedTime: Date | null;
+}) {
+  const { narmesteLederId } = useParams<{ narmesteLederId: string }>();
+
+  const [isPendingFerdigstillPlan, startFerdigstillPlanTransition] =
+    useTransition();
+
+  const [
+    { isLastUtkastSaveSuccess, utkastLastSavedTime },
+    saveUtkastDispatchAction,
+    isSavingUtkast,
+  ] = useActionState(saveUtkastAction, {
+    isLastUtkastSaveSuccess: true,
```

Comment:
hmz, skal denne være alltid true? Det var ikke åpenbart for meg hva denne er til, eller hvorfor den har hardkodet at det alltid er success forrige gang(?)